src/admin/components/UserList/PermissionGroup.js

```diff
@@ -0,0 +1,136 @@
+import React, {Fragment, useState} from 'react';
+import PropTypes from 'prop-types';
+import {List, Icon, Grid, Accordion} from 'semantic-ui-react';
+import {permissionIcon, permissionColor} from '../../../common/enums';
+
+/**
+ * Display permission groups
+ */
+const PermissionGroup = ({groupOptions}) => {
+  const [activeIndex, setActiveIndex] = useState(0);
+  return (
+    <Accordion fluid styled>
+      {groupOptions &&
+        groupOptions.map(group => (
+          <Fragment key={group.key}>
+            <Accordion.Title
+              active={activeIndex === group.key}
+              index={group.key}
+              onClick={titleProps =>
+                setActiveIndex(activeIndex === group.key ? '' : group.key)
+              }
+              data-testid="user-group-header"
+            >
+              <Icon name="dropdown" />
+              {group.key}
+            </Accordion.Title>
+            <Accordion.Content active={activeIndex === group.key}>
+              {group.key === 'Administrators' ? (
+                <Grid columns={3}>
+                  <Grid.Column>
```

Comment:
Instead of repeating `<Grid.Column>` the same way each time, why not make a component that takes a list of permissions and returns the column so it doesn't need to be repeated?